app/src/docs/content/core/introToCore.mdx

```diff
@@ -36,7 +36,7 @@ Four of the main views offer create functions; Streams, Canvases, Dashboards and
 In the case of canvases, these are built within the powerful Core visual programming editor. For more on this see Working with Canvases. Dashboards are also a unique case, because dashboards are basically view-only versions of the visualisation modules from canvases, so they have their own UI allowing you to add modules from your canvases and arrange the dashboard tiles sizes and layout. For more, see Working with Dashboards.
 
 ### User profile
-You can set or edit your profile image (useful in future iterations of the Marketplace), username, email address  and ethereum address by either going to the Settings menu (at upper right with your initials) or by clicking on your profile image.
+You can set or edit your profile image (useful in future iterations of the Marketplace), username, email address and ethereum address by either going to the Settings menu (at upper right with your initials) or by clicking on your profile image.
```

Comment:
ethereum = Ethereum?

Reply:
Instead of initials, we will use the ETH address (or username for legacy accounts):
![Screenshot 2020-11-23 at 13 59 37](https://user-images.githubusercontent.com/1064982/100978204-bb3b2680-354a-11eb-826a-64b4afb3bf90.png)